components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -6457,7 +6461,7 @@ def reexportedName(name):
                 haveUnscopables = True
                 cgThings.append(
                     CGList([CGGeneric("const unscopable_names: &[&[u8]] = &["),
-                            CGIndenter(CGList([CGGeneric(str_to_const_array(name)) for
+                            CGIndenter(CGList([CGGeneric(str_to_byte_slice(name)) for
```

Comment:
We need to change the type above and fix https://github.com/servo/servo/blob/1c6b74e1f16f6c92f80410d98608064fa4fbede2/components/script/dom/bindings/interface.rs#L241